contracts/fio.system/src/voting.cpp

```diff
@@ -1225,12 +1232,16 @@ namespace eosiosystem {
             //check(!isproxy || !pitr->proxy, "account that uses a proxy is not allowed to become a proxy");
             if (isproxy && !pitr->proxy) {
                 votersbyowner.modify(pitr, same_payer, [&](auto &p) {
+                    p.fioaddress = fio_address;
+                    p.addresshash = addresshash;
                     p.is_proxy = isproxy;
                     p.is_auto_proxy = false;
                 });
             }else if (!isproxy) { //this is how we undo/clear a proxy
                 name nm;
                 votersbyowner.modify(pitr, same_payer, [&](auto &p) {
+                    p.fioaddress = "";      // TODO: placed here to revert to same state as from regproducer, not sure if correct, verify...
+                    p.addresshash = 0;      // TODO: placed here to revert to same state as from regproducer, not sure if correct, verify...
```

Comment:
We okay with this TODO here?

Reply:
I don't see the need for it. I placed it in my code cleanup queue.